### ISSUE TITLE
SabreDAV returns 'Not Found' exception for all DAV URLs

### ISSUE THREAD

**Issue by dantefromhell** [KeudellCoding/HumHub-Modules-HumDAV#2]
I installed HumDAV version 1.3.2 on the free cloud offering from HumHub, it runs on version 1.10.3

For the CardDAV and CalDAV URLs I both get the following error when calling them directly from my browser

```
  | <?xml version="1.0" encoding="utf-8"?>
-- | --
  | <d:error xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns">
  | <s:sabredav-version>3.2.2</s:sabredav-version>
  | <s:exception>Sabre\DAV\Exception\NotFound</s:exception>
  | <s:message>Principal with name <EMAIL_REDACTED> not found</s:message>
  | </d:error>

```

The URLs mentioned in the README file, all return the following DAV error
```
<?xml version="1.0" encoding="utf-8"?>
--
  | <d:error xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns">
  | <s:sabredav-version>3.2.2</s:sabredav-version>
  | <s:exception>Sabre\DAV\Exception\NotFound</s:exception>
  | <s:message>Node with name 'personal' could not be found</s:message>
  | </d:error>
```

There are no errors in the HumHub application log, besides the warnings about '.htaccess' not writable.

**Comment by dantefromhell** [comment]
@KarlKeu00 
To clarify, the fix for this issue was released with version 1.4.0?

**Comment by KarlKeu00** [comment]
I hope so :)
